templates/sources.html

```diff
@@ -183,21 +187,35 @@ <h3 id="relationship-name"></h3>
             });
             source.data('id', data[0].id);
             applyRules(data[0].rules);
-            source.html(data[0].name);
+            source.text(data[0].name);
             populateRelationships(data[0].relationships);
         }
         stream('Traits can be used inside ability commands as variables, which get replaced at runtime with their corresponding values.');
         restRequest('POST', {'index':'sources', 'id': $('#profile-source-name').val()}, loadSourceCallback);
     }
 
     function addFactRow(trait, value){
-        $('#factTbl tbody').append(
-            '<tr>' +
-                '<td><p contenteditable="true">' + trait + '</p></td>' +
-                '<td><p contenteditable="true">' + value + '</p></td>' +
-                '<td><p onclick="$(this).closest(\'tr\').remove()">&#x274C;</p></td>'+
-            '</tr>'
-        )
+        $('#factTbl tbody').append($("<tr></tr>")
+            .append($("<td></td>")
+                .append($("<p></p>")
+                    .text(trait)
+                    .attr("contenteditable", "true")
+                )
+            )
+            .append($("<td></td>")
+                .append($("<p></p>")
+                    .text(value)
+                    .attr("contenteditable", "true")
+                )
+            )
+            .append($("<td></td>")
+                .append($("<p></p>")
+                    .html("&#x274C;")
+                    .attr("contenteditable", "true")
+                    .attr("onclick", "$(this).closest('tr').remove()")
```

Comment:
Could this use jQuery's `.click()` function? Something like (hoping this arrow function works right):

```
... .click(event => {
    $(event.currentTarget).closest('tr').remove();
});
```